src/6model/reprs/P6bigint.c

```diff
@@ -8,41 +8,41 @@
 static MVMint64 mp_get_int64(MVMThreadContext *tc, mp_int * a) {
     MVMuint64 res;
     MVMuint64 signed_max = 9223372036854775807ULL;
-
-    if (a->used == 0) {
-        return 0;
-    }
+    int bits = mp_count_bits(a);
 
     /* For 64-bit 2's complement numbers the positive max is 2**63-1
      * but the negative max is 2**63. */
     if (MP_NEG == SIGN(a)) {
+        if (bits > 64) {
```

Comment:
Ok. Since it confused me, it will probably confuse other causal readers, so can you please extend the comment to make clear that bits will be 64 for -2^63?